OPHD/States/MapViewStateTurn.cpp

```diff
@@ -106,15 +106,8 @@ void MapViewState::updatePopulation()
 	int nurseries = structureManager.getCountInState(Structure::StructureClass::Nursery, StructureState::Operational);
 	int hospitals = structureManager.getCountInState(Structure::StructureClass::MedicalCenter, StructureState::Operational);
 
-	// FOOD CONSUMPTION
-	int food_consumed = mPopulation.update(mCurrentMorale, mFood, residences, universities, nurseries, hospitals);
 	StructureList &foodproducers = structureManager.structureList(Structure::StructureClass::FoodProduction);
-	int remainder = food_consumed;
-
-	if (mFood > 0)
-	{
-		remainder -= pullFood(mFood, remainder);
-	}
+	int remainder = mPopulation.update(mCurrentMorale, mFood, residences, universities, nurseries, hospitals);
```

Comment:
The removal of the `if` block doesn't coincide with the commit message. Maybe double check that the removal was intentional.

Reply:
It was, as per a previous comment, I accidentally squashed too many commits into one. The if block is incorrect in this context as `MapViewState::mFood` is a cached value, not a working pool. The actual food storage is in the structures themselves.